Found the following flake8 issues in code after latest rebase from master

Here is list from pr 563 a few minutes ago.  However, on my local system these issues do not show up.
Strange.  This list came from the appvuyer output of the failed test of the pr.

setup.py:537:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/cim_xml.py:90:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/cim_http.py:121:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/mof_compiler.py:269:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/mof_compiler.py:282:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/mof_compiler.py:289:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/mof_compiler.py:305:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/mof_compiler.py:1909:1: E305 expected 2 blank lines after class or function definition, found 1
pywbem/_recorder.py:79:1: E305 expected 2 blank lines after class or function definition, found 1
testsuite/test_cim_http.py:350:1: E305 expected 2 blank lines after class or function definition, found 1
testsuite/run_cim_operations.py:3659:1: E305 expected 2 blank lines after class or function definition, found 1
testsuite/run_cim_operations.py:3886:1: E305 expected 2 blank lines after class or function definition, found 1
testsuite/validate.py:86:1: E305 expected 2 blank lines after class or function definition, found 1
13    E305 expected 2 blank lines after class or function definition, found 1
make: *** [makefile:316: flake8.log] Error 1
ERROR: InvocationError: 'C:\\cygwin\\bin\\make.EXE check'
___________________________________ summary ___________________________________
ERROR:   pywin: commands failed

The only think I do not understand is why this was not caught on the pr where the changes were made.

**COMMENT KS:** Identified the problem.  flake8 just updated from 3.0.4 (at least that is what I had) to 3.2.0 and apparently added test for 2 spaces at end of class definition.  Tested the changes and the old flake8 does not complain so filing pr after doing tests.

The only thing I do not understand is why when I create a new python virtual env and install pywbem it still brings up flake8 version 3.0.4 and have to do the pip install flake8 --upgrade.

--- a/docs/mof_compiler.help.txt
+++ b/docs/mof_compiler.help.txt
@@ -84,5 +84,5 @@ General options:
   -v, --verbose         Print more messages while processing
   -h, --help            Show this help message and exit
 
-Example: mof_compiler CIM_Schema_2.45.mof -s https://localhost:15989 -n
-root/cimv2 -u sheldon -p penny42
+Example: mof_compiler CIM_Schema_2.45.mof -s https://localhost -n root/cimv2
+-u sheldon -p p42

--- a/os_setup.py
+++ b/os_setup.py
@@ -165,16 +165,18 @@ import pip
 
 
 # Some types to avoid dependency on "six" package during installation.
+# pylint: disable=invalid-name
 if sys.version_info[0] == 2:
-    string_types = basestring,      # pylint: disable=invalid-name # noqa: F821
-    text_type = unicode             # pylint: disable=invalid-name # noqa: F821
-    binary_type = str               # pylint: disable=invalid-name # noqa: F821
+    string_types = basestring,      # noqa: F821
+    text_type = unicode             # noqa: F821
+    binary_type = str               # noqa: F821
     from xmlrpclib import ServerProxy as xmlrpc_ServerProxy
 else:
-    string_types = str,             # pylint: disable=invalid-name # noqa: F821
-    text_type = str                 # pylint: disable=invalid-name # noqa: F821
-    binary_type = bytes             # pylint: disable=invalid-name # noqa: F821
+    string_types = str,             # noqa: F821
+    text_type = str                 # noqa: F821
+    binary_type = bytes             # noqa: F821
     from xmlrpc.client import ServerProxy as xmlrpc_ServerProxy
+# pylint: enable=invalid-name
 
 
 class OsDistribution(Distribution):

